src/db/collections/analytics/address_balance.rs

```diff
@@ -1,47 +1,22 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use async_trait::async_trait;
 use futures::TryStreamExt;
 use mongodb::bson::doc;
 use serde::{Deserialize, Serialize};
 
-use super::{Analytic, Error, Measurement, PerMilestone};
+use super::Error;
 use crate::{
-    db::{collections::OutputCollection, MongoDb, MongoDbCollectionExt},
-    types::{stardust::milestone::MilestoneTimestamp, tangle::MilestoneIndex},
+    db::{collections::OutputCollection, MongoDbCollectionExt},
+    types::tangle::MilestoneIndex,
 };
 
-/// Computes the number of addresses that hold a balance.
-#[derive(Debug)]
-pub struct AddressAnalytics;
-
 #[derive(Copy, Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[allow(missing_docs)]
 pub struct AddressAnalyticsResult {
     pub address_with_balance_count: u64,
 }
 
-#[async_trait]
-impl Analytic for AddressAnalytics {
-    async fn get_measurement(
-        &mut self,
-        db: &MongoDb,
-        milestone_index: MilestoneIndex,
-        milestone_timestamp: MilestoneTimestamp,
-    ) -> Result<Option<Measurement>, Error> {
-        db.collection::<OutputCollection>()
-            .get_address_analytics(milestone_index)
-            .await
-            .map(|measurement| {
-                Some(Measurement::AddressAnalytics(PerMilestone {
-                    milestone_index,
-                    milestone_timestamp,
-                    inner: measurement,
-                }))
-            })
-    }
-}
-
 impl OutputCollection {
```

Comment:
We should think about removing the old analytics queries/impl because they won't be used anymore. I'm afraid that both implementations will drift apart over time and introduce weird bugs. @Alex6323 wdyt?

Reply:
#1080